pkg/ansible/proxy/cache_response.go

```diff
@@ -141,11 +142,37 @@ func (c *cacheResponseHandler) ServeHTTP(w http.ResponseWriter, req *http.Reques
 }
 
 // skipCacheLookup - determine if we should skip the cache lookup
-func (c *cacheResponseHandler) skipCacheLookup(r *requestfactory.RequestInfo) bool {
+func (c *cacheResponseHandler) skipCacheLookup(r *requestfactory.RequestInfo, gvk schema.GroupVersionKind, req *http.Request) bool {
 	// check if resource is present on request
 	if !r.IsResourceRequest {
 		return true
 	}
+	owner, err := getRequestOwnerRef(req)
+	if err != nil {
+		log.Error(err, "Could not get owner reference from proxy.")
+		return false
+	}
+	ownerGV, err := schema.ParseGroupVersion(owner.APIVersion)
+	if err != nil {
+		m := fmt.Sprintf("Could not get group version for: %v.", owner)
+		log.Error(err, m)
+		return true
+	}
```

Comment:
Could we not do the same whiout check if the GKV exist in the cluster? 
I mean, is required to do the check? 

Because if a dev, would like to add in the blacklist, for example, a third-party API which may exist or not it will not work. So, since we are comparing strings I think may we could not add these checks.

Reply:
I don't think this is checking the cluster, I think it's just parsing the `apiVersion` string and returning a `GroupVersion` struct: https://godoc.org/k8s.io/apimachinery/pkg/runtime/schema#ParseGroupVersion

Reply:
It is true ignore my comment .. tks.